### ISSUE TITLE
影响终端连接服务器

### ISSUE THREAD

**Issue by starsyi** [yangfeicheung/Shadowsocks-X#30]
我使用的是macOS HighSierra 10.13.3上的 ShadowsocksX 2.13(23)
当我使用shadowsocks-X之后，我使用终端ssh连接远程服务器，会自动断开，提示connection to * closed by remote host。
当我关闭shadowsocks-X后，ssh登录到远程服务器，没有任何影响，工作正常。
终端工具iterm2和自带的终端都是一样的报错，断开连接。

**Comment by yangfeicheung** [comment]
Yup, high sierra changed ssh, and it makes use of system proxy setting, so if you enable app, you need to note that by default proxy has a timeout value of 10, that's why ssh keeps getting blocked.

**Comment by yangfeicheung** [comment]
@starsyi
hi bro, this issue is fixed in macOS 10.13.4,  if you upgrade to this version, the issue won't exist anymore, seems it's macOS's ssh issue.
